packages/app-admin-ui/client/components/Nav/NavIcons.js

```diff
@@ -15,6 +15,10 @@ export function NavIcons() {
   let { graphiqlPath, signoutPath, authStrategy } = useAdminMeta();
   return ENABLE_DEV_FEATURES || authStrategy ? (
     <NavGroupIcons>
+      <NavIcon href="/" title="Home">
```

Comment:
This is inside the `ENABLE_DEV_FEATURES` flag which means it will only be shown in dev and in future only when this flag is explicitly set.

Was that intentional?